infra/core/src/main/java/com/simprints/core/tools/extentions/Flow.ext.kt

```diff
@@ -0,0 +1,53 @@
+package com.simprints.core.tools.extentions
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.drop
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.flow.scan
+
+fun <T1, T2, T3, T4, T5, T6, T7, T8, T9, R> combine9(
```

Comment:
This is getting out of hand. Can you add a TODO comment/ticket to review ways to simplify this in future? Potentially, by doing some nested combining of parts of the state instead.

Reply:
As I understand, it doesn't go over 5 because it becomes harder. While there is nothing inherently wrong with combining 9 flows in one large function, there should be a way to split it into several smaller (more focused) sub-state combinations purely for maintainability.